GRSlib/motion/scoring.py

```diff
@@ -1,17 +1,35 @@
-#from GRSlib.parallel_tools import ParallelTools
+from GRSlib.Ver0_Files.opt_tools import internal_generate_cell, get_desc_count, *
+from GRSlib.parallel_tools import ParallelTools
 #from GRSlib.motion.lossfunc.moments import Moments
 #from GRSlib.motion.lossfunc import Gradient
 from GRSlib.converters.sections.lammps_base import Base, _extract_compute_np
+from examples.self_test.GRS_protocol import GRSModel, GRSSampler
 import lammps, lammps.mliap
 from lammps.mliap.loader import *
 from functools import partial
-import numpy as np
-
+import numpy as vnp
+from GRSlib.GRS import *
 #Scoring has to be a class within motion because we want a consistent reference for scores, and this
 #refrence will be LAMMPS using a constructed potential energy surface from the representation loss function.
 #Sub-classes of Scoring will be versions of this representation loss function (Moments, Entropy, etc), allowing
 #for custom verions to be added without trouble.
 
+n_totconfig = 10
```

Comment:
Dont want to bury inputs deep in the code, use the io method and pass the read in config to the functions that need it.